src/constants/TabList.tsx

```diff
@@ -0,0 +1,45 @@
+import CategoriesIcon from '@/assets/icons/menu-hamburger.svg?react';
+import DefaultDiscoverIcon from '@/assets/icons/dashboard-default.svg?react';
+import ActiveDiscoverIcon from '@/assets/icons/dashboard-filled.svg?react';
+import DefaultHomeIcon from '@/assets/icons/home-default.svg?react';
+import ActiveHomeIcon from '@/assets/icons/home-filled.svg?react';
+import DefaultChatIcon from '@/assets/icons/chat-default.svg?react';
+import ActiveChatIcon from '@/assets/icons/chat-filled.svg?react';
+import DefaultMyIcon from '@/assets/icons/person-default.svg?react';
+import ActiveMyIcon from '@/assets/icons/person-filled.svg?react';
+import { RouterPath } from '@/routes/path';
+
+const TabList = [
```

Comment:
특정한 형태로 작성되어야 한다면 여기는 typescrtip를 통해 타입을 지정하면 좋을거 같아요 
```suggestion
// ex)
const TabList: { lable, defaailicont ... } " { ... } = [
```